data/add_categories_to_db.py

```diff
@@ -0,0 +1,21 @@
+import sqlite3
+import os
+from dotenv import load_dotenv
+
+categories_fixtures = ["на голову", "на вуха", "на шию", "на руки", "аксесуари", "набори"]
+
+load_dotenv()
+basedir = os.path.abspath('.')
+
+
+db_path = os.path.join(basedir, 'data', 'app.db')
+
+conn = sqlite3.connect(db_path)
+cursor = conn.cursor()
+
+
+for value in categories_fixtures:
+    cursor.execute('INSERT INTO categories (category_name) VALUES (?)', (value,))
```

Comment:
wrap into try/except block, because we will include unique constraint

Reply:
Done